### DISCUSSION TITLE
RustDesk firewall and session features improvement

### DISCUSSION THREAD

**Discussion by adispy** [rustdesk/rustdesk#2646] [Ideas]
Hi community,
Since I am a big fan of OpenSource I wanted to try this software since I need something non-commercial for privacy and be able to stay connected without being kicked out. After I tested RustDesk, I have identified some features that can be improved (sorry if they were discussed before):
- Running on port 443. This is a big issues, since in most corporate environments the actual RustDesk ports are blocked. If this will be corrected in a future version, it will be awesome! This will also make the product increase in popularity since it will be a big competitor with the commercial ones.
- Install version. If I install RustDesk (tested on Windows), when I connect from a remote endpoint I can see that the session is acting like an RDP one. I cannot see the other computer's desktop even tough it is logged in, I get the logon screen. This defeats the product main purpose of screen sharing.

Anyway...great product and hope it will stay with us

Regards,

**Comment by jmlindemann** [comment]
I came here to make the same request of port 443. Without this optional setting, it severely limits the functionality /usability of this app. As it is right now, I can't use this on my work computer due to their firewall rules.  But port 80/443 are standard ports anywhere.

**Comment by thermafensterbau** [comment]
+1 here.
ATM we are considering Rustdesk self hosted in our infrastructure, and for support of our customers.
Using "special" TCP and UDP ports will cause a lot of hassle on customers computers, because you never ever know their strange network configs.
Even in our networks it is a problem, we have to reconfigure our firewalls, but its solvable.
